wire-compiler/src/main/java/com/squareup/wire/schema/WireRun.kt

```diff
@@ -225,3 +225,7 @@ data class WireRun(
     return result
   }
 }
+
+fun List<ProtoFile>.link(): Schema {
```

Comment:
Do we care about Java callers invoking this? Right now they would call `WireRunKt.link(foo)` which isn't great.

Also I'm not sure whether this should have a more descriptive name like `toSchema` or `linkToSchema`.

Reply:
The closest example in the stdlib is probably `Iterable<Pair<K, V>>.toMap` so I think I've vote for `toSchema` for Kotlin callers.

But for Java I'd probably want `Schema.fromFiles(files)`.